### ISSUE TITLE
ESC selection problem

### ISSUE THREAD

**Issue by ahfynjj** [dgatf/msrc#145]
After selecting FUTABA SBUS2 as the receiver protocol and the esc protocol as hobbywing V4/flyfun, select the ESC model such as platinum60A, update config, disconnect, and reconnect. The platinum60A selected in the model will disappear.

**Comment by dgatf** [comment]
Yes, that's normal because the HW ESC model it is not stored, but its parameters yes. This is because those parameters changes from model to model (HW V4 and Flyfun) and I don't have the full list of HW models witht the correct parameters. When you select the HW ESC model, msrc_gui proposes parameters values based on users feedback. So, parameters are passed to msrc and you can adjust them in case you need to do so. Unfortunately those HW ESCs provides voltage and current in raw values.
